### PR TITLE
Enable compute optimizer infra deployments from all regions

### PR DESCRIPTION
**Pull request by jeff-zinger**
In response to #877 , we are removing the requirement of only deploying the infrastructure for the compute optimizer from the us-west-2 region.

### CODE REVIEW

**Review by dfratte — APPROVED** [review]
I've tested this against regions in the US and EU. It deploys fine. Waiting for maintainer's approval before merge.

**Review by ccasher — APPROVED** [review]

**Pull request merged**
